js/src/forum/utils/getCleanDisplayName.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Whether to use the old mentions format.
+ *
+ * `'@username'` or `'@"Display name"'`
+ */
+export const shouldUseOldFormat = () => app.forum.data.attributes.allowUsernameMentionFormat || false;
```

Comment:
Should this use `app.forum.attribute()`? Not sure why we'd access the data directly.

Reply:
I have no clue either... Fixed it now.